can-component.js

```diff
@@ -178,8 +177,13 @@ function makeReplacementTagCallback(tagName, componentTagData, shadowTagData, le
 			// We need to teardown any bindings created too so we create a nodeList
 			// to do this.
 
+
+
 			var nodeList = nodeLists.register([el],
-				tagData.teardown || noop,
+				function() {
+					var fn = tagData.teardown || noop;
+					fn();
+				},
```

Comment:
What does this do differently than passing `tagData.teardown || noop` to `nodeLists.register`?

Reply:
Nothing, just gives you a place to set a breakpoint.

Reply:
It doesn't.  That function was in there for another reason but not necessary now.  Should be reverted back to the previous.